crmsh/report/constants.py

```diff
@@ -3,7 +3,7 @@
 BIN_CRM = "/usr/sbin/crm"
 BIN_COLLECTOR = f"{BIN_CRM} report __collector"
 COMPRESS_DATA_FLAG = "COMPRESS CRM_REPORT DATA:::"
-LOG_PATTERNS = "CRIT: ERROR: error: warning: crit:"
+LOG_PATTERNS = "CRIT: ERROR: WARNING: crit: error: warning:"
```

Comment:
Instead of defining these patterns as space-separated string, we can define it directly as a list of strings.